Jenkinsfile

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env groovy
+
+@Library('lco-shared-libs') _
+
+pipeline {
+	agent any
+	environment {
+		dockerImage = null
+		PROJ_NAME = projName("${JOB_NAME}")
+		GIT_DESCRIPTION = gitDescription()
+		DOCKER_IMG = dockerImageName("${LCO_DOCK_REG}", "${PROJ_NAME}", "${GIT_DESCRIPTION}")
+	}
+	stages {
+		stage('Build image') {
+			steps {
+				script {
+					dockerImage = docker.build("${DOCKER_IMG}")
+				}
+			}
+		}
+		stage('Push image') {
+			steps {
+				script {
+					dockerImage.push("${GIT_DESCRIPTION}")
+				}
+			}
+		}
+		stage('Test') {
+		    steps {
+		        script {
+                    sh 'docker run --rm "${DOCKER_IMG}" -w=/lco/banzai/ --user=root python setup.py test'
```

Comment:
`-w=/lco/banzai/` can be removed, because it looks like that's specified in the Dockerfile.